### ISSUE TITLE
Is there any plan of rescoping this project ?

### ISSUE THREAD

**Issue by PrateekBhat2** [openfoodfacts/community-portal#86]

**Comment by M123-dev** [comment]
I don't think so @PrateekBhat2 , we now have a new forum where discussions can be held. But I was never a part of the decision making of this project

**Comment by alexgarel** [comment]
@PrateekBhat2  this project might be open to new specs :-) 
It's not in the list of our current priorities !

cc: @teolemon

**Comment by PrateekBhat2** [comment]
okay np! just was curious about this :)

**Comment by Sudhanva-Nadiger** [comment]
If it's opening again pls start in typescript and go pls

**Comment by teolemon** [comment]
@PrateekBhat2 
At the latest iteration, it was almost becoming a social network of its own (vs the forum/slack). We're not going to rescope it/restart it.
- We're going to implement a Moderation Tool first, with possibly a commenting feature.
- We're going to improve the Open Food Facts profile page

@Sudhanva-Nadiger 
If you're looking for a JS project: https://github.com/openfoodfacts/taxonomy-editor
If you're looking for a Go project: https://github.com/openfoodfacts/openfoodfacts-go
